packages/imageviewer-extension/src/index.ts

```diff
@@ -159,6 +177,23 @@ function addCommands(app: JupyterLab, tracker: IImageTracker) {
 
     if (widget) {
       widget.scale = 1;
+      widget.rotation = 0;
+    }
+  }
+
+  function rot90(): void {
+    const widget = tracker.currentWidget;
+
+    if (widget) {
+      widget.rotation += 90;
```

Comment:
I don't think it ultimately matters too much, but I like to take the modulus of angles by 360 so that they are clearer when inspected.

Reply:
Thanks for the comments @ian-r-rose! I'll have to figure out why initial scale is changing, but the rest are no-brainers.

Reply:
Removing max-width: 100% and max-height: 100% in the CSS seems to fix the scaling issue. I don't know if that breaks something else, but I don't see any problems so far.